overlord/overlord.go

```diff
@@ -93,6 +101,36 @@ func loadState(backend state.Backend) (*state.State, error) {
 	return state.ReadState(backend, r)
 }
 
+// Run runs a loop to ensure the current state regularly through StateEngine Ensure().
+func (o *Overlord) Run() {
+	intv := ensureInterval
+	o.loopTomb.Go(func() error {
+		tim := time.NewTimer(intv)
```

Comment:
Can we call this `timer`? Doesn't feel like the two letters save enough, and tim reads like a person's name to me. :)